database/migrations/2023_11_19_060120_report_templates.php

```diff
@@ -70,6 +87,14 @@ public function up(): void
             ],
+            [
+                'template_name' => 'scnae_tb',
+                'template' => '{"14": ["3 01 01 010","3 01 01 020"],"22": [["4 02 01 040","4 02 01 980","4 02 01 990","4 02 02 010","4 02 02 020","4 02 02 050","4 02 02 220","4 02 02 990","4 04 02 010","4 04 02 020","4 06 01 010"],["5 01 01 010","5 01 01 020","5 02 01 010","5 02 01 020","5 02 02 010","5 02 03 010","5 02 03 020","5 02 03 070","5 02 03 080","5 02 03 090","5 02 03 990","5 02 04 010","5 02 04 020","5 02 05 010","5 02 05 020","5 02 05 030","5 02 05 040","5 02 10 030","5 02 11 030","5 02 11 990","5 02 12 020","5 02 12 030","5 02 13 040","5 02 13 050","5 02 13 060","5 02 13 070","5 02 16 020","5 02 16 030","5 02 99 010","5 02 99 020","5 02 99 030","5 02 99 050","5 02 99 060","5 02 99 070","5 02 99 990","5 05 01 010","5 05 01 050","5 05 01 060","5 05 01 070","5 05 01 090","5 05 01 990","5 03 01 040","5 05 03 060","5 05 04 990"],["4 03 01 020"]],"25": ["3 01 01 010","3 01 01 020"]}',
+            ],
+            [
+                'template_name' => 'scbaa_tb',
+                'template' => '{"17":["4 02 01 040","4 02 01 980","4 02 01 990"],"18":["4 02 02 010","4 02 02 020","4 02 02 050","4 02 02 220","4 02 02 990"],"19":["4 06 01 010"],"23":["4 04 02 010","4 04 02 020"],"29":["5 01 01 010","5 01 01 020"],"30":["5 02 01 010","5 02 01 020","5 02 02 010","5 02 03 010","5 02 03 020","5 02 03 070","5 02 03 080","5 02 03 090","5 02 03 990","5 02 04 010","5 02 04 020","5 02 05 010","5 02 05 020","5 02 05 030","5 02 05 040","5 02 10 030","5 02 11 030","5 02 11 990","5 02 12 020","5 02 12 030","5 02 13 040","5 02 13 050","5 02 13 060","5 02 13 070","5 02 16 020","5 02 16 030","5 02 99 010","5 02 99 020","5 02 99 030","5 02 99 050","5 02 99 060","5 02 99 070","5 02 99 990"],"24":["4 03 01 020"],"41":[["4 02 01 040","4 02 01 980","4 02 01 990","4 02 02 010","4 02 02 020","4 02 02 050","4 02 02 220","4 02 02 990","4 04 02 010","4 04 02 020","4 06 01 010"],["5 01 01 010","5 01 01 020","5 02 01 010","5 02 01 020","5 02 02 010","5 02 03 010","5 02 03 020","5 02 03 070","5 02 03 080","5 02 03 090","5 02 03 990","5 02 04 010","5 02 04 020","5 02 05 010","5 02 05 020","5 02 05 030","5 02 05 040","5 02 10 030","5 02 11 030","5 02 11 990","5 02 12 020","5 02 12 030","5 02 13 040","5 02 13 050","5 02 13 060","5 02 13 070","5 02 16 020","5 02 16 030","5 02 99 010","5 02 99 020","5 02 99 030","5 02 99 050","5 02 99 060","5 02 99 070","5 02 99 990","5 05 01 010","5 05 01 050","5 05 01 060","5 05 01 070","5 05 01 090","5 05 01 990","5 03 01 040","ckdj: Loss on Sale of Property, Plant & Equipment","5 05 03 060","5 05 04 990"],["4 03 01 020"]]}'
```

Comment:
pa-double check lang mamaya may mga na double ata brackets and may mga nasama na di dapat kabilang sa template